zerver/tests/tests.py

```diff
@@ -542,6 +542,11 @@ def test_add_bot(self):
             event['event']
         )
 
+        result = self.client.get('/json/users')
```

Comment:
the `mypy` errors from static analysis are because you've changed the type of `result` from what it was used for above.  Should just pick a different variable name for the second usage.